src/ukify/ukify.py

```diff
@@ -747,6 +781,13 @@ def parse_args(args=None):
     p.add_argument('--secureboot-certificate',
                    dest='sb_cert',
                    help='path to certificate file or engine-specific designation for SB signing')
+    p.add_argument('--secureboot-certificate-dir',
+                   dest='sb_certdir',
+                   default='/etc/pki/pesign',
+                   help='path to nss certificate database directory for PE signing. Default is /etc/pki/pesign')
+    p.add_argument('--secureboot-certificate-nick',
+                   dest='sb_cert_nick',
+                   help='nickname of nss certificate database entry to use for PE signing')
```

Comment:
The help texts should have `[sbsign]`/`[pesign]` added to them to signal that you don't have to pass all of them, or this gets really confusing. Also, please call this `--secureboot-certificate-name`.

Or reuse `--secureboot-certificate` and add an explicit `--secureboot-signtool=` or so.

Reply:
it's common to have both installed, so a -signtool= option probably makes sense

Reply:
I think an option that takes either sbsign (default) or pesign looks nicer, but it's not too important

Reply:
added `--signtool`